recipes/otsvm/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_DOC=OFF \
+  ..
+
+make install
+DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib ctest -R pyinstallcheck -E saveload --output-on-failure ${MAKEFLAGS}
```

Comment:
When I asked this (as @patricksnape is asking now), the rule has been to have a single terminal newline at the end of each file.